src/single-spa-angular-cli.js

```diff
@@ -114,15 +113,16 @@ const unloadTag = (url) => {
   };
 };
 
-const bootstrap = (opts) => {
+const bootstrap = (opts, props) => {
+  const {singleSpa} = props
```

Comment:
single-spa is now being passed in via props, as of https://github.com/CanopyTax/single-spa/releases/tag/v3.9.0